### ISSUE TITLE
Could provide full command for your script

### ISSUE THREAD

**Issue by tlz4320** [klarman-cell-observatory/ivPerturbSeq#1]
In your pipeline, there are a lot of params, which I can't infer from your paper.
Therefore, if someone want to reuse your code, have to read your data, and construct the parameters like "pert".
So, could your support your full command to readme or other new script, for others better use of your method.

**Comment by seanken** [comment]
Hey! Yeah, happy to add in a bit more detail on that--will try to get to it in the next week or so. Thanks for bringing it up!

**Comment by tlz4320** [comment]
> Hey! Yeah, happy to add in a bit more detail on that--will try to get to it in the next week or so. Thanks for bringing it up!

Thank you very very much. And sorry to seek advice for an another thing. I downloaded the GEO data from GEO database which you upload your pertub-sequencing result. But I found that data in GEO of single-cell only have one end of sequencing data. But in your metadata, noted should have paired sequencing. Therefore, is there any other platform that could I get the whole data from it. In the end, thank your for rapidly replying.

**Comment by seanken** [comment]
Yeah, GEO handles 10X data a little funny. For GEO I uploaded the bam files from 10X instead of the fastqs (as suggested by the 10X website), so to get the data from GEO/SRA you need to download the original data (which you can get to on the associated SRA run page, under the access data tab). If you want to get the paired fastq files you can make it from the bam (10X has a tool called bamtofastq to do just that, I believe). See more detail at: https://kb.10xgenomics.com/hc/en-us/articles/360024716391-What-format-of-10x-Genomics-data-should-I-submit-to-NCBI-GEO-
